salt/journal/config/srv-journal-docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+
+services:
+    redis:
```

Comment:
Won't be needed in `prod`?

Reply:
The formula however runs a build that won't create an ElastiCache (it takes longer). I think we can swap it in and out depending on whether ElastiCache is present.